components/Picker.jsx

```diff
@@ -0,0 +1,221 @@
+/**
+ * Picker.jsx
+ * Created by Kevin Li 8/16/17
+ */
+
+import React, { createRef, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+require('../styles/components/_picker.scss');
+
+const fontAwesomeIconId = "usa-dt-picker__button-icon--svg";
+
+const propTypes = {
+    sortFn: PropTypes.func,
+    icon: PropTypes.string,
+    altText: PropTypes.string,
+    iconColor: PropTypes.string,
+    backgroundColor: PropTypes.string,
+    iconSize: PropTypes.string,
+    selectedOption: PropTypes.string,
+    className: PropTypes.string,
+    id: PropTypes.string,
+    options: PropTypes.arrayOf(PropTypes.shape({
+        name: PropTypes.string,
+        onClick: PropTypes.func
+    })),
+    borderType: PropTypes.oneOf(['none', 'bottom', 'full']),
+    dropdownDirection: PropTypes.oneOf(['left', 'right']),
+    borderColor: PropTypes.string,
+    isFixedWidth: PropTypes.bool,
+    children: PropTypes.node
+};
+
+const getBorderType = (borderType, borderColor) => {
```

Comment:
Do we want to include this? Could we just use our styles?

Reply:
You can just use `borderType="none"` and apply whatever css to design your own border.